server/log_rpc_server_test.go

```diff
@@ -1206,15 +1206,15 @@ func TestGetEntryAndProofSkewNoProof(t *testing.T) {
 
 	response, err := server.GetEntryAndProof(context.Background(), &getEntryAndProofRequest17_11)
 	if err != nil {
-		t.Fatalf("get entry and proof should have succeeded but we got: %v", err)
+		t.Errorf("get entry and proof should have succeeded but we got: %v", err)
```

Comment:
That one probably should still be `Fatal` because `response` is likely to be nil :)

Reply:
If response is nil, this will crash :-)

Reply:
but ok